.travis.yml

```diff
@@ -90,6 +90,12 @@ matrix:
     script: ./travis.sh
     after_script: ccache -s # show ccache stats
 
+  - os: linux
+    name: "Transifex push template Ubuntu 16.04 Xenial"
+    sudo: required
+    dist: xenial
+    script: bash autobuild/transifex-push-translations.sh
```

Comment:
Also i suggest to change shell inside `transifex-push-translations.sh` from `/bin/sh` to `/bin/bash`, as `pushd/popd` cannot be executed by `sh`